core/invoker/src/main/scala/whisk/core/container/HttpUtils.scala

```diff
@@ -50,7 +50,7 @@ import whisk.core.entity.size.SizeLong
  * @param timeoutMsec the timeout in msecs to wait for a response
  * @param maxResponse the maximum size in bytes the connection will accept
  */
-protected[container] class HttpUtils(
+class HttpUtils(
```

Comment:
`whisk.core.containerpool.docker.DockerContainer` uses `HttpUtils` so it cannot be `protected[container]`. Changed to `protected[core]`.